Automattic/wp-calypso
# Add real-time comments to Reader full-post view

We're getting our Calypso ["sea legs"](https://www.merriam-webster.com/dictionary/sea%20legs) back under us. And very open to detailed integration input as this gets off the ground. 

1. Make it work. 
2. Make it beautiful. <--here.
3. Make it fast (if you need to, "90% of the time" step two has you covered).

#### Changes proposed in this Pull Request

* Add [Phoenix](https://github.com/phoenixframework/phoenix)-based websocket middleware.
* Add real-time comments to Reader full-post view behind a feature flag.
* Disable Reader comment polling while we explore the space.

#### Testing instructions

1. Apply D40881-code to your sandbox.
2. Fire up this branch locally or [hit calypso.live](https://calypso.live/?branch=try/realtime-comments).
3. Go to a full post view in the Reader _on a public post_. View in one window and comment in another. Comment from traditional WP.com. Comment from Notifs.

You should see the Reader update real-time in all cases.
